### DISCUSSION TITLE
Basis behind SequentialFeatureSelector: k_features = 'parsimonious'

### DISCUSSION THREAD

**Discussion by jievqgrg** [rasbt/mlxtend#962] [Q&A] [answered]
I wonder what is the basis for the above methodology (see http://rasbt.github.io/mlxtend/api_subpackages/mlxtend.feature_selection/#sequentialfeatureselector). The motivation behind it is clear to me. In fact, I am very excited to learn about this methodology because I have a problem where it is a potential solution (see https://stats.stackexchange.com/questions/585203/feature-subset-selection-decision-rule).

But can someone explain to me what the theoretical basis for it is? :)

**Comment by rasbt** [comment, marked as answer]
Like you suggested, it is more based on common sense rather than a theoretical basis 😅. Usually, "within 1 standard error" can be considered as a negligible difference here. It's more of a rule of thumb, but I think I read this somewhere once though. If I had to guess, it was Elements of Statistical Learning by Hastie et al.

**Comment by jievqgrg** [comment]
Thank you very much! You are right, I found it in Hastie et. al., The Elements of Statistical Learning, 2nd Edition: p. 61, 244 (Corrected 12th printing - Jan 13, 2017).
